bin/check_dependencies

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -uo pipefail
```

Comment:
Why no `-e`?

Reply:
With it the script would die after the docker run or after diff